content/api/users/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Endpoint: users”
+tags: ["API"]
+category: "api"
+menu:
+  api:
+    identifier: users
+    weight: 200
+---
+
+## /users/add
+
+```http
+POST /api/v1/users/add.ext HTTP/1.1
+Host: https://yoursite.vanillaforums.com
+```
+
+Insert a user.
+
+[__Authentication__](../#making-api-calls): required  [more](http://docs.vanillaforums.com/api/)
+
+### Parameters
+
+| Parameter   | Type                                  | Description                             |
+| ----------- | ------------------------------------- | --------------------------------------- |
+| __`Name`__  | `string`                              | Username                                |
+| __`Email`__ | `string`                              | Email address                           |
+| `Password`  | `string`                              | User’s password                         |
+| `Photo`     | `String`                              | Photo URL                               |
+| `ShowEmail` | `boolean`                             | Allow others to see this email address? |
+| `Gender`    | `string`                              | `m`,`f`, or `u` for unspecified         |
+| `Title`     | `string`                              | Freeform user title                     |
+| `Location`  | `string`                              | Freeform user location                  |
+| `Role`      | [`multi smart id`](../multi smart-id) | Role ID (corresponds to GDN_Role)       |
+| `Rank`      | [`smart id`](../smart-id)             | Rank ID (corresponds to GDN_Rank)       |
+
+### Notes
+
+The Photo field submitted here is only able to take a remotely hosted URL, not a file upload. In order to upload an avatar to the service, use the [/users/photo API](http://docs.vanillaforums.com/api/users/)
```

Comment:
Your links are broken here. Don't use the production domain name. Hugo has a way to reference internal pages:

https://gohugo.io/content-management/cross-references/